libs/application/templates/financial-aid/src/fields/Summary/Files.tsx

```diff
@@ -25,14 +26,17 @@ const Files = ({
   personalTaxReturn,
   taxFiles,
   incomeFiles,
+  childrenFiles,
   applicationId,
 }: Props) => {
   const { formatMessage } = useIntl()
   const [createSignedUrlMutation] = useMutation(CreateSignedUrlMutation)
 
+  const concatAllFiles = [...incomeFiles, ...taxFiles, ...childrenFiles]
+
   const allFiles = !personalTaxReturn
-    ? incomeFiles.concat(taxFiles)
-    : incomeFiles.concat(taxFiles).concat([personalTaxReturn])
+    ? concatAllFiles
+    : concatAllFiles.concat([personalTaxReturn])
```

Comment:
Nit, but I would personally go for this:

const allFiles = [...incomeFiles, ...taxFiles, ...childrenFiles, ...(personalTaxReturn ? [personalTaxReturn] : [])]